crates/sui-graphql-rpc/src/types/transaction_block.rs

```diff
@@ -32,7 +37,28 @@ pub(crate) struct TransactionBlock {
     pub gas_input: Option<GasInput>,
     #[graphql(skip)]
     pub epoch_id: Option<u64>,
-    pub kind: Option<TransactionBlockKind>,
+    // pub kind: Option<TransactionBlockKind>,
+}
+
+#[ComplexObject]
```

Comment:
that's weird, im p sure this code is in main already. have you rebased on main yet?